gulpfile.js

```diff
@@ -10,42 +10,60 @@ var del = require('del');
 
 require('babel-core/register');
 
-var src = [
-  'src/shady-css/common.js',
-  'src/shady-css/node-factory.js',
-  'src/shady-css/node-visitor.js',
-  'src/shady-css/stringifier.js',
-  'src/shady-css/token.js',
-  'src/shady-css/tokenizer.js',
-  'src/shady-css/parser.js',
-  'src/*.js'
+var src = ['src/**/*.js'];
+var dist = 'dist';
+var concatOrder = [
+  '/shady-css/common.js',
+  '/shady-css/node-factory.js',
+  '/shady-css/node-visitor.js',
+  '/shady-css/stringifier.js',
+  '/shady-css/token.js',
+  '/shady-css/tokenizer.js',
+  '/shady-css/parser.js',
+  '/*.js'
+].map(function(path) {
+  return 'src' + path;
+});
+var measureable = [
+  dist + '/shady-css.concat.js',
+  dist + '/shady-css.min.js'
 ];
-var dest = 'dist/';
 
 gulp.task('default', ['clean', 'test', 'build', 'minify', 'measure']);
 
-gulp.task('build', ['test'], function() {
+gulp.task('build', ['test', 'transform', 'minify']);
+
+gulp.task('concat', ['test'], function() {
+  return gulp.src(concatOrder)
+    .pipe(babel())
+    .on('error', function(error) {
+      console.error(error);
+    })
+    .pipe(concat('shady-css.concat.js'))
```

Comment:
Rollup looks good. By not exporting the 'internal' modules, it looks like it can save the space of module definitions for them.